src/screens/ExploreApps/ExploreAppsItem.js

```diff
@@ -0,0 +1,78 @@
+// @flow
+/*
+    Pillar Wallet: the personal data locker
+    Copyright (C) 2019 Stiftung Pillar Project
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+import * as React from 'react';
+import { Image, Linking } from 'react-native';
+import styled from 'styled-components/native';
+import { MediumText, BaseText } from 'components/Typography';
+import { fontStyles } from 'utils/variables';
+import type { AppItem } from 'utils/exploreApps';
+import Button from 'components/Button';
+import { themedColors } from 'utils/themes';
+
+interface Props {
+  item: AppItem
+}
```

Comment:
Just to make it consistent with other screens:
```suggestion
type Props = {
  item: AppItem,
}
```